ReservacionDeVuelos/src/screens/FinishScreen/index.js

```diff
@@ -10,40 +10,41 @@ import UserContext from '../../context/UserContext';
 
 const FinishScreen = ({navigation, route}) => {
   const [changeStyle, setChangeStyle] = useState(true);
-  const [addData, setAddData] = useState(true);
+  const [addData, setAddData] = useState(false);
   const data = route.params.values;
 
   const user = React.useContext(UserContext);
 
-  if (addData === true) {
-    firestore()
-      .collection('flights')
-      .add({
-        uid: user.user.uid,
-        avDestino: data.to.title,
-        avOrigen: data.from.title,
-        destino: data.to.subtitle,
-        fecha: data.date,
-        origen: data.from.subtitle,
-        passangers: data.passengers,
-      })
-      .then(() => {
-        console.log('User added!');
-      })
-      .catch(function (error) {
-        console.log(
-          'There has been a problem with your fetch operation: ' +
-            error.message,
-        );
-        // ADD THIS THROW error
-        throw error;
-      });
-  }
-
+  useEffect(() => {
+    console.log(addData);
```

Comment:
Remove the Console.log()

Reply:
Ty for your observation, I forgot that console.log